### DISCUSSION TITLE
Search in MPCORB

### DISCUSSION THREAD

**Discussion by deepsky-moers** [mworion/MountWizzard4#309] [Q&A] [answered]
Hello all,
I tried to find the minor planet Artemis in MW4, but I could not find the planets in the selection. In think the search is done in only in parts of the major list MPCORB.
Artemis is in MPCORB. I there a way to find this panet in any selction in MW4?

thanks.
Peter

**Comment by mworion** [comment, marked as answer]
Hi Peter,
actually I do not offer the full file as the download > 150 MB takes too long.
I hope I will find a solution which better fits. Sorry about now
Michel

**Comment by mworion** [comment]
Peter,
as I'm working bringing an online search to Minor Planet Center: Could you describe the use cases you need ?
Michel

**Comment by deepsky-moers** [comment]
Dear Michel, just to find an Object in MPcorb, to transfer it to the mount.
Like the Minor Planet „Artemis“ or so.
Of course is can feed the elements in to the mount by an simple text file to import by the Updater Programm.
That’s all.
With my best regards 
Peter

Von meinem iPhone gesendet
